designer/client/components/Visualisation/Lines.tsx

```diff
@@ -57,10 +57,21 @@ export class Lines extends React.Component<Props, State> {
               <g key={pointsString}>
                 <polyline
                   onClick={() => this.editLink(edge)}
+                  onKeyPress={(event) =>
+                    event.key === "Enter" || event.key === " "
+                      ? this.editLink(edge)
+                      : null
+                  }
```

Comment:
could you add this as a class method please (like how it's done for onClick, `onClick={() => this.editLink(edge)}`). It will just clean up the markup and make things easier to follow.

 I suppose you could even change editLink to accept event and edge. so both onClick and onKeyPress would look like this `(event) => this.editLink(event, edge)`?

Reply:
Done! I decided against making editLink accept `event` in the end - accepting both MouseEvent and KeyboardEvent made the types a bit untidy (maybe just me being unfamiliar with TypeScript)

Reply:
Make sense. Thanks!